conf/defaults.ini

```diff
@@ -393,6 +393,7 @@ headers =
 enabled = false
 config_file = /etc/grafana/ldap.toml
 allow_sign_up = true
+sync_cron = @hourly
```

Comment:
I'd add a comment explaining that this is related to ldap sync in enterprise.

Something like: `# LDAP backround sync (Enterprise)`

Reply:
Oh, I definitely want to do that! Was thinking to add such comment with the rest of the documentation, but I can do it in this pull right away :)